src/module.jl

```diff
@@ -2,8 +2,6 @@
 # Some file utils
 # –––––––––––––––
 
-VERSION > v"0.4-" && (beginswith = Base.startswith)
```

Comment:
Just as a cleanup, there are a few places where `Base.startswith` can be unqualified